standard/lexical-structure.md

```diff
@@ -1025,7 +1025,7 @@ right_shift_assignment
 
 ### 6.5.1 General
 
-The pre-processing directives provide the ability to conditionally skip sections of compilation units, to report error and warning conditions, and to delineate distinct regions of source code.
+The pre-processing directives provide the ability to conditionally skip sections of compilation units, to report error and warning conditions, to delineate distinct regions of source code, and to set the nullable context.
```

Comment:
```suggestion
The pre-processing directives provide the ability to conditionally skip sections of compilation units, to report error and warning conditions, to delineate distinct regions of source code, and to set the nullable contexts.
```

(Given that we're using the plural form later.)

Alternatively, we could decide we have a single nullable context, which has properties for both warnings and annotations.